app/src/org/commcare/tasks/SendTask.java

```diff
@@ -83,34 +87,55 @@ protected Boolean doTaskBackground(Void... params) {
 
             publishProgress(Localization.get("bulk.send.dialog.progress", new String[]{"" + (i + 1)}));
 
-            File f = files[i];
+            File formFolder = files[i];
 
-            if (!(f.isDirectory())) {
-                Log.e("send", "Encountered non form entry in file dump folder at path: " + f.getAbsolutePath());
-                CommCareApplication._().reportNotificationMessage(NotificationMessageFactory.message(StockMessages.Send_MalformedFile, new String[]{null, f.getName()}, MALFORMED_FILE_CATEGORY));
+            if (!(formFolder.isDirectory())) {
+                Log.e("send", "Encountered non form entry in file dump folder at path: " + formFolder.getAbsolutePath());

```

Comment:
`"send"` -> `TAG`
(might need to add `private static final String TAG = SendTask.class.getSimpleName();`)